src/modules/4-clear.js

```diff
@@ -0,0 +1,7 @@
+export default function clearTasks() {
+  /* Clear completed tasks */
+
+  const parsedArr = JSON.parse(localStorage.getItem('taskArr'));
+  const temp = parsedArr.filter((item) => item.completed === false);
```

Comment:
A function for the "Clear all completed" button using the `Array.filter()` method is implemented 🐐 Awesome!